test/CommonGlobalVarNoAddrspace.ll

```diff
@@ -0,0 +1,24 @@
+; Ensure "common global" with no addrspace is reported as an error
```

Comment:
quite unused directory, but should be placed here IMHO

test/CommonGlobalVarNoAddrspace.ll -> test/negative/CommonGlobalVarNoAddrspace.ll

Reply:
done